src/workspaces/common/requester-pays/RequesterPaysModal.tsx

```diff
@@ -6,8 +6,7 @@ import { FieldsArg } from 'src/libs/ajax/workspaces/providers/WorkspaceProvider'
 import { FormLabel } from 'src/libs/forms';
 import * as Nav from 'src/libs/nav';
 import { requesterPaysProjectStore } from 'src/libs/state';
-import * as Utils from 'src/libs/utils';
-import { DEFAULT } from 'src/libs/utils';
+import { cond, DEFAULT } from 'src/libs/utils';
```

Comment:
removing import * from Utils here, since it's better and more traceable to call-out the explicit things needed from the package.